ui/admin-portal/src/app/components/job/view/uploads/view-job-uploads/view-job-uploads.component.ts

```diff
@@ -74,6 +74,7 @@ export class ViewJobUploadsComponent implements OnInit {
     this.saving = true;
     this.jobService.updateJobLink(this.job.id, docType, updateLinkRequest).subscribe(
       job => {
+        this.job = job;
```

Comment:
This shouldn't be necessary, should it? The job will get updated by the emit, which will updated the parent job, which then gets input into this component.